src/app/legacy/containers/CpsRecommendations/RecommendationsPromo/__snapshots__/index.test.jsx.snap

```diff
@@ -237,6 +237,5 @@ exports[`RecommendationsPromo it renders a Story Promo wrapped in a Grid compone
       </div>
     </div>
   </div>
-  ,
 </div>
```

Comment:
It must have come from within the `shouldMatchSnapshot` code previously? As long as it's not required by screenreaders as a way of pausing, then it might be OK? But I would definitely like to get to the bottom of why it's like this (it's not obvious from your code changes in `src/app/legacy/containers/CpsRecommendations/RecommendationsPromo/index.jsx` so it must be something in `src/app/legacy/containers/CpsRecommendations/RecommendationsPromo/index.test.jsx` instead?